symfony/plugins/orangehrmCorePlugin/Api/V2/RequestParams.php

```diff
@@ -154,7 +154,7 @@ public function getBooleanOrNull(string $type, string $key, ?bool $default = nul
         if (is_null($param)) {
             return null;
         }
-        return $this->$type->getBoolean($key, $default);
+        return $this->$type->getBoolean($key, $param);
```

Comment:
@devishke-orange please check

Reply:
@RajithaKumara  looks good 👍. we made this same change in the performance branch